### ISSUE TITLE
How to rename file?

### ISSUE THREAD

**Issue by catouse** [mysticatea/cpx#33]
I want to copy a icon file to a new place and rename it to favicon.ico.

```js
cpx.copy('./resources/icon.ico', `${destRoot}/favicon.ico`)
```

**Comment by rendomnet** [comment]
Any news?

**Comment by jhildenbiddle** [comment]
I'd also like to see the ability to rename files added.

**Comment by entozoon** [comment]
Yeah, it's annoying that you can't do this. Same with [copyfiles](https://github.com/calvinmetcalf/copyfiles/)
However, [cpy-cli](https://github.com/sindresorhus/cpy-cli) does have this feature
